src/cryptonote_protocol/cryptonote_protocol_handler.h

```diff
@@ -51,7 +51,8 @@ PUSH_WARNINGS
 DISABLE_VS_WARNINGS(4355)
 
 #define LOCALHOST_INT 2130706433
-#define CURRENCY_PROTOCOL_MAX_OBJECT_REQUEST_COUNT 500
+#define CURRENCY_PROTOCOL_MAX_OBJECT_REQUEST_COUNT 100
```

Comment:
I bet `500` blocks limit works pretty well. Any good reason to change this?

Reply:
To mitigate DoS potential, as above. 100 objects are smaller than 500 (ceteris paribus).